vignettes/getting_started_with_LundTaxR.Rmd

```diff
@@ -78,7 +79,7 @@ sjodahl_classes <- classify_samples(
 ## Exploring Classification Results
 
 The `classify_samples()` function returns a comprehensive list object containing multiple components. Let's explore the different types of results and understand what information is available:
-
+%% seems like you get info for 267 samples out of 307 in the dataset - maybe mention that?
```

Comment:
The paragraph above should be updated to say `267 samples` instead of `307 samples`. Feel free to update this.